Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM golang:1.16-alpine3.13 as builder
+
+WORKDIR /app
+COPY . .
+RUN apk add --no-cache make ca-certificates
+RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -o gql_server -ldflags="-w -s" ./app/graphql_server/main.go
+
+FROM scratch
```

Comment:
Those are two images in one file, right? (I've never seen two `FROM` blocks in one file.)